waku/v2/node/config.nim

```diff
@@ -129,6 +129,11 @@ type
       defaultValue: ""
       name: "filternode" }: string
     
+    filterTimeout* {.
+      desc: "Timeout for filter node in seconds.",
+      defaultValue: 86400
```

Comment:
Maybe add a comment to say this translates to `24h`. Was this value chosen arbitrarily, or do we have good reason to believe this is a reasonable default? My own gut-feeling would be a default of ~4h, which is more or less the max that most mobile networks allow devices to be offline before forcing them to reregister. Of course, some applications (especially with high message rates) may set this to something much, much shorter.

Reply:
The time was arbitary, changed the same to 4h in commit bfc3104ca29a499cc4512975ba8e90e7f6071342